### PR TITLE
[WIP] update readme

### LINKED ISSUE #149: Improve README.me
- [x] add Github description
- [x] add repo logo
- [x] improve links to docs, website, etc..

### PR DESCRIPTION
**Pull request by antho1404**
fix #149

### CODE REVIEW

**Comment by antho1404** [comment]
test comment

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
+![MESG core](https://cdn.discordapp.com/attachments/450115250881036348/451705186856075265/MESG-Github-ban3-02.jpg)
```
[the remark this answers is not in the document]
Comment (NicolasMahe) [reply README.md]: And we should use the SVG version!

[the remark this answers is not in the document]
Comment (NicolasMahe) [reply README.md]: Done

[the remark this answers is not in the document]
Comment (antho1404) [reply README.md]: between https://github.com/mesg-foundation/core/blob/c2afb7b80068507f08b6b02c567e4b9071081907/README.md and https://github.com/mesg-foundation/core/blob/40e9e23bf80398ab04cec043e0f79fd990bb0498/README.md I really prefer the one where it's not huge and right in the middle and also I think jpg is fine for this after yes we can re-upload it directly to github but anyway github does that even with external url anyways

**Comment by NicolasMahe** [comment]
We decided with Will to move most of you addition to a new page get started on the documentation website.
We should keep the README.md of this repo about the development of MESG-Core, not its utilization (the docs.mesg.tech is dedicated to this).

**Pull request merged**